### ISSUE TITLE
Posgresql user password appears in clear in capistrano.log

### ISSUE THREAD

**Issue by turgu1** [capistrano-plugins/capistrano-postgresql#55]
When a new user is created in the postgresql database, the password appears in clear in the log file and at the screen during deployment. It would be better to hide it as it could be considered a security breach if someone get access to the log file or  looking at the screen over the shoulder.

It think this is a feature to be added in the future. Or a new parameter that can request hiding the create user command from the log. A warning in the readme file would also be appropriate. My 2 cents...

Guy

**Comment by NorseGaud** [comment]
Yep, I agree. It's a consequence of using the new execute instead of the test command. Curious what others think about this too.

**Comment by NorseGaud** [comment]
Feature request to wrap a section of execute's arguments to hide them in the STDOUT and logs: https://github.com/capistrano/sshkit/issues/429

**Comment by NorseGaud** [comment]
Submitted https://github.com/capistrano/sshkit/pull/430 for review of the feature

**Comment by NorseGaud** [comment]
So close to getting this feature into sshkit. Once it's in, I'll release 5.1.0 with the feature.

**Comment by NorseGaud** [comment]
https://github.com/capistrano/sshkit/pull/430 merged into master. Waiting on next release sometime next week.

**Comment by NorseGaud** [comment]
Releasing in 5.1.0
